woocommerce/woocommerce
# New task list loading placeholder for experiment 1

### All Submissions:






### Changes proposed in this Pull Request:

Made two different changes:
- Some general changes to improve the loading time of the task list
- Make use of the Two column task list placeholder for the first task list experiment 

![task-list-placeholder-experiment1](https://user-images.githubusercontent.com/2240960/161610845-1efd31ed-3669-496c-b9e1-214db2585681.gif)


### How to test the changes in this Pull Request:

1. On a new site finish the onboarding wizard
2. Load the home screen
3. It should show a new task list placeholder that actually shows a header as part of the loading design (see GIF as well)
4. Task list should display and load before the `notes` have been loaded below

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### FOR PR REVIEWER ONLY:
